### ISSUE TITLE
Custom elements seems to expect a name

### ISSUE THREAD

**Issue by AndreasHeintze** [h5o/h5o-chrome#10]
If I have custom tags in my HTML where the name of the tag starts with nav, then HTML5 outliner seems to think this is a nav element and therefore it expects a name to be set. This must be wrong, isn't it?

```
<nav-bar>
    <nav-megamenu></nav-megamenu>
    <nav-megamenu></nav-megamenu>
    <nav-megamenu></nav-megamenu>
</nav-bar>
```

<img width="516" alt="skarmavbild 2015-08-02 kl 13 25 42" src="https://cloud.githubusercontent.com/assets/2219544/9025928/1b41f924-391a-11e5-8238-ad524de70223.png">

**Comment by dominykas** [comment]
The outliner checks for the `tagName` with a regex: https://github.com/h5o/h5o-js/blob/master/src/utils.js#L51 - the regex does have a `^` and a `$` in there to do an exact (not a partial) match, so I wonder - how do you construct these elements? Are you extending `nav`? What's the actual `tagName` that you see in dev tools for them?

**Comment by AndreasHeintze** [comment]
Here is a quick and simple example:
http://fiddle.jshell.net/AndreasHeintze/z7hm3xdm/2/show/

I'm using RiotJS.

**Comment by dominykas** [comment]
Thanks, will take a look.

**Comment by dominykas** [comment]
Fixed in 0.8.13 - available via direct install (see releases here) or as soon as Google approve it on web store.

**Comment by AndreasHeintze** [comment]
Thanks!
